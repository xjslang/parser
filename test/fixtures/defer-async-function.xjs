async function asyncTest() {
  console.log('async function start')
  defer {
    console.log('async function defer')
  }
  console.log('async function end')
}

asyncTest()
