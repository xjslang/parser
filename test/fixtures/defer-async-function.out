async function start
async function end
async function defer
